File: MocaTemplate/MocaItem/SeleniumPage/DefaultPage.vb

Imports OpenQA.Selenium
Imports MiYABiS.SeleniumTestAssist

Public Class $safeitemname$
    Inherits SeleniumAction

    Public Overrides ReadOnly Property MyPageName As String
        Get
            Return "Default.aspx"
        End Get
    End Property

    Public Sub New(ByVal driver As IWebDriver, ByVal baseUrl As String)
        MyBase.New(driver, baseUrl)
    End Sub

End Class

File: MocaTemplate/MocaItem/SeleniumPage/DefaultPage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using OpenQA.Selenium;
using MiYABiS.SeleniumTestAssist;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace $rootnamespace$
{
    class $safeitemname$ : SeleniumAction
    {

        public $safeitemname$(IWebDriver driver, string baseUrl) : base(driver, baseUrl)
        {
            
        }

        public override string MyPageName
        {
            get
            {
                return "Default.aspx";
            }
        }

    }
}
